package.json

```diff
@@ -122,6 +123,7 @@
     "*/*/src/**/*.{js,ts}": "eslint --fix"
   },
   "dependencies": {
-    "@types/ssh2": "0.5.46"
+    "@types/ssh2": "0.5.46",
+    "artillery": "^1.7.2"
```

Comment:
@travis-payne Please lock it down to an exact version (e.g. remove the caret).
https://github.com/hyperledger/cactus/blob/main/CONTRIBUTING.md#on-reproducible-builds